frontend/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -1,6 +1,199 @@
+/* eslint-disable react/no-unescaped-entities */
 import React from 'react';
+import drag from './Assets/img/drag-and-drop.png';
+import flag from './Assets/img/flag.png';
+import padlock from './Assets/img/open-padlock.png';
+import view from './Assets/img/view.png';
+import S from './styles';
 
-// TODO: Setup landing page
-const LandingPage = () => <div>LandingPage</div>;
+const LandingPage = () => (
+  <S.LandingPageContainer>
+    <div>
+      <svg
+        width="850"
+        height="546"
+        viewBox="0 0 1386 546"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        style={{ position: 'absolute' }}
+        transform="translate(300 100)"
+      >
+        <path
+          d="M367.575 56.6426C-95.7607 17.8601 -48.9425 377.275 117.508 498.025C375.556 685.224 607.177 254.691 1033.21 391.582C1459.25 528.473 1466.06 3.24546e-05 1230.8 0C995.552 -3.24546e-05 819.324 94.4553 367.575 56.6426Z"
+          fill="#EFDBFE"
+        />
+      </svg>
+      <svg
+        width="900"
+        height="546"
+        viewBox="0 0 1462 546"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        style={{ position: 'absolute' }}
+        transform="translate(220 40)"
+      >
+        <path
+          d="M387.493 57.0908C-100.95 18.3438 -51.5946 377.43 123.875 498.069C395.908 685.097 640.079 254.958 1089.2 391.724C1538.33 528.49 1545.5 0.500032 1297.5 0.5C1049.5 0.499968 863.722 94.8689 387.493 57.0908Z"
+          fill="#E5C2FF"
+        />
+      </svg>
+      <S.LandingPageTitle>Introducing our features</S.LandingPageTitle>
+      <S.CardContainer>
+        <S.Card>
```

Comment:
Can we abstract the Card component out and make it reusable and just have an items array to map over to show cards?

We could have a `FeaturesCard` component that takes in the title, img, text content and start/end colors for example.